.github/workflows/post-release.yml

```diff
@@ -78,9 +78,9 @@ jobs:
           MANIFEST_SHA=${MANIFEST_SHA#"sha256:"}
           COSIGN_PASSWORD=${{ secrets.COSIGN_PASSWORD }} cosign sign -r \
             -key <(echo -n "${{ secrets.COSIGN_PRIVATE_KEY }}" | base64 --decode) \
-            -a tag=${{ env.LIFECYCLE_VERSION }} \
-            buildpacksio/lifecycle:sha256-${MANIFEST_SHA}.sbom
-          cosign verify -key cosign.pub -a tag=${{ env.LIFECYCLE_VERSION }} buildpacksio/lifecycle:sha256-${MANIFEST_SHA}.sbom
+            -a tag=${{ env.LIFECYCLE_VERSION }} -attachment sbom \
+            buildpacksio/lifecycle:${{ env.LIFECYCLE_VERSION }}@${MANIFEST_SHA}
+          cosign verify -key cosign.pub -a tag=${{ env.LIFECYCLE_VERSION }} -attachment sbom buildpacksio/lifecycle:${{ env.LIFECYCLE_VERSION }}
```

Comment:
I can't make a code suggestion because the lines didn't change, but lines 34, 38, and 42 should use `jq -r .[0].critical.image.\"docker-manifest-digest\"` instead of `jq -r .critical.image.\"docker-manifest-digest\"` because cosign 1.2.0 returns the verification as an array.